packages/carbon-web-components/tools/get-rollup-config.js

```diff
@@ -129,6 +130,17 @@ function getRollupConfig({
         include: [/node_modules/],
         sourceMap: true,
       }),
+      minifyHTMLLiterals({
+        failOnError: true,
+        options: {
+          minifyOptions: {
+            caseSensitive: true,
+            collapseInlineTagWhitespace: true,
+            collapseWhitespace: true,
+            removeComments: true,
+          },
+        },
+      }),
```

Comment:
There's a section below that is supposed to minify the html templates here: 
https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/tools/get-rollup-config.js#L198-#L216

I believe kenny put a comment in saying it wasn't working, so maybe we can remove that redundant code and replace it with this plugin?